src/components/paragraph/index.jsx

```diff
@@ -1,9 +1,33 @@
 import PropTypes from "prop-types";
 
-const Paragraph = ({ children }) => <p className="c-paragraph">{children}</p>;
+const COMPONENT_CLASS_NAME = "c-paragraph";
+
+const Paragraph = ({ children, className, truncationLines }) => (
+	<p
+		data-style-truncation-lines={truncationLines > 0 ? "true" : "false"}
+		style={{
+			WebkitLineClamp: truncationLines > 0 ? truncationLines : "none",
```

Comment:
Any reason why this is not using a CSS Variable to allow more flexibility?

Reply:
Doh, honestly I am going mental

Reply:
@matthewroach Added line clamp into the token system.  Also it's a good example cause it's a little tricky on how many dashes needed when applying it in React.

Reply:
@matthewroach Updated code based on what you did in the meeting.  Tests also modified to comply to the changes.